lib/puppet/provider/posix_acl/posixacl.rb

```diff
@@ -89,15 +92,18 @@ def check_set
   end
 
   # TODO: Investigate why we're not using this parameter
-  def permission=(_value)
-    Puppet.debug @resource.value(:action)
+  def permission=(value)
+    Puppet.debug "resource.value(:action): #{@resource.value(:action)}"
+    Puppet.debug "value parameter for permission method is: #{value}"
+    Puppet.debug "new permissions from the instance are: #{@resource.value(:permission)}"
```

Comment:
Are these still needed?

Reply:
I thought they are quite helpful during debugging and it might help people if they see issues in the future?